src/controllers/shortcode.controller.ts

```diff
@@ -8,6 +8,9 @@ export async function createRandomShortCode(longUrl: string): Promise<ShortCode>
     const shortCode = intToRadix64(randomCode)
 
     // TODO: retry if exists
+    if(await getShortCodeDetails(shortCode)) {
```

Comment:
remove the TODO comment, and instead add a comment that says that if the shortcode exists, we will recurse.

Reply:
I have removed the TODO and added the desired comment as well.